docs/validations/validations-rules-reference.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: 'Rules Reference'
+---
+
+# Rules Reference
+
+Rules can be asserted against a gameplay recording to ensure that the recording meets specific criteria. We provide a set of rule types that can be used to construct assertions based on various performance metrics.
+
+Rules belong to a Validation Suite, and are asserted each time the Validation Suite is run.
+
+## Rule Types
+
+### Frames Per Second (with percentile)
+
+Assert that the frames per second of the gameplay recording are above a threshold, within a specified percentile.
+
+### Average Frames Per Second
+
+Assert that the average frames per second of the gameplay recording is above a threshold.
```

Comment:
I think we need to clarify how this average is calculated. i.e. whether this is an average over the entire gameplay recording or whether this is a rolling average over the last N ticks/frames.

Reply:
thanks. Added